vm/vm.go

```diff
@@ -143,6 +143,10 @@ type VM struct {
 	stop  chan struct{}
 }
 
+func (vm *VM) ImmutableState(_ context.Context) (state.Immutable, error) {
+	return vm.State()
+}
```

Comment:
This seems to return a reference directly to merkledb, can we specify a single trie view of the last accepted state?

Reply:
I can create an issue for this

Reply:
resolved with your PR https://github.com/ava-labs/hypersdk/pull/1543